apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx

```diff
@@ -49,22 +81,426 @@ export const getPageInfo: GetPageNavInfo = ({
+                    <p
+                      data-h2-font-weight="base(700)"
+                      data-h2-margin-bottom="base(x.5)"
+                    >
+                      {response.screeningQuestion?.question
+                        ? response.screeningQuestion.question[locale]
+                        : ""}
+                    </p>
+                    <p>{response.answer}</p>
+                  </li>
+                ))}
+              </ul>
+            </div>
+          ) : (
+            <Well>
+              <p data-h2-text-align="base(center)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "<strong>It looks like you haven't answered any screening questions yet.</strong>",
+                  id: "Zx7wZQ",
+                })}
+              </p>
+            </Well>
+          )}
+        </ReviewSection>
+      )}
+
+      <section data-h2-margin-bottom="base(x3)">
+        <Heading level="h4" data-h2-margin="base(0, 0, x1, 0)">
+          {intl.formatMessage({
+            defaultMessage: "Sign and submit",
+            id: "+sWqVY",
+          })}
+        </Heading>
+        <div>
+          <FormProvider {...methods}>
+            <form onSubmit={methods.handleSubmit(handleSubmit)}>
+              <p data-h2-margin-bottom="base(x1)">
+                {intl.formatMessage({
+                  defaultMessage: `"You made it! By signing your name, you confirm that:"`,
+                  id: "d0KeTK",
+                })}
+              </p>
+              <ul>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage: `"I’ve reviewed everything written in my application"`,
+                      id: "IFVdRD",
+                    })}
+                  </p>
+                </li>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage: `"I understand that I am part of a community who trusts each other"`,
+                      id: "MKU7g2",
+                    })}
+                  </p>
+                </li>
+                <li>
+                  <p data-h2-margin-bottom="base(x.5)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "I promise that the information I’ve provided is true",
+                      id: "kAMYLw",
+                    })}
+                  </p>
+                </li>
+              </ul>
+              <div>
+                <Input
+                  id="signature"
+                  label={intl.formatMessage({
+                    defaultMessage: "Please type your name",
+                    id: "DmeN52",
+                    description:
+                      "Label displayed for signature input in sign and submit page.",
+                  })}
```

Comment:
This label seems more like instructions rather than labelling the input. I think we should keep labels short, clear and descriptive. This text seems more like what we would add to a `aria-description` attribute.

REF: https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions#:~:text=A%20field%20for%20entering%20a%20given%20name%20is%20clearly%20labeled%20with%20%22Given%20Name%22%20and%20the%20field%20for%20family%20name%20is%20labeled%20%22Family%20Name%22%20to%20avoid%20confusion%20over%20which%20name%20is%20requested.

@mnigh Would "Name" be a more appreciate label?

Reply:
I agree.

Reply:
Talked to Josh and he suggested using "Your full name". That should work correct?

Reply:
Changed to "Your full name" in [20852b4](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/20852b41b5811088012cf5769ecce51b0866997d)